lessServer.js

```diff
@@ -62,11 +62,11 @@ function compileLessFile(res, lessFileContents, lessFilePath, cssFilePath, paths
                      if (compilationError) {
                          // Log Paths to help with debuggin - often an issue with include precedence
                          console.log("PATHS (in order of precedence)): \n" + paths.split(',').join("\n"));
-                         less.writeError(compilationError);
+                         console.error(compilationError);
                          return reqErr(res, compilationError, "LESS PARSER ERROR: Line " + compilationError.line + ', ' + compilationError.message + '. File: ' + lessFilePath);
                      }
                      if (compileToFile) {
-                         writeCompiledOutputToCssFile(res, compiledCssOutput, cssFilePath);
+                         writeCompiledOutputToCssFile(res, compiledCssOutput.css, cssFilePath);
```

Comment:
The `.css`, is that a typo?

Reply:
no, there seems to be a css string in the object https://worldsurfleague.slack.com/archives/GN7PETRTJ/p1585580998031600?thread_ts=1585092046.135400&cid=GN7PETRTJ

Reply:
I see. In that case, can we lock in the version for less?
```
RUN npm install less
```

We might run into other future incompatibilities.

Reply:
👍